lib/tasks/addon-install.js

```diff
@@ -34,22 +34,23 @@ module.exports = Task.extend({
 
     ui.startProgress(chalk.green('Installing addon package'), chalk.green('.'));
 
+    var willSave = this.project.isEmberCLIAddon() && commandOptions.save;
     return npmInstall.run({
       packages: packageNames,
-      'save-dev': true,
+      'save-dev': !willSave,
       'save-exact': true
     }).then(function() {
       return this.project.reloadAddons();
     }.bind(this)).then(function() {
-      return this.installBlueprint(blueprintInstall, packageNames, blueprintOptions);
+      return this.installBlueprint(blueprintInstall, packageNames);
```

Comment:
What's going on here? I don't know how `blueprintOptions` was consumed, but this feels like it may be breaking.

/cc @trabus